### PR TITLE
Doc:add translation for zh_CN

### PR DESCRIPTION
**Pull request by xuezhixin**

### CODE REVIEW

**Comment by kgaillot** [comment]
Hi @xuezhixin , we are about to release version 2.1.7. If you resubmit this against the 2.1 branch instead of main, I should be able to get it into the final release, otherwise it will become part of the next release in mid-2024.

File: po/zh_CN.po [hunk 1]
```diff
@@ -234,6 +234,23 @@ msgstr ""
 "如果shutdown-lock为true，并且将此选项设置为非零持续时间，则自从开始shutdown以"
 "来经过了这么长的时间后，shutdown锁将过期，即使该节点尚未重新加入。"
 
+#: daemons/controld/controld_control.c:683 lib/pengine/common.c:164
+msgid ""
+"How long to wait for a node that has joined the cluster to join the "
+"controller process group"
+msgstr ""
+"已加入集群的节点加入控制器进程组等待的时间"
```
Comment (gao-yan) [inline po/zh_CN.po:242]: To be more clear, it could be something like:
"等待已加入集群的节点加入控制器进程组的时间".

File: po/zh_CN.po [hunk 2]
```diff
@@ -234,6 +234,23 @@ msgstr ""
...
+
+#: daemons/controld/controld_control.c:685 lib/pengine/common.c:166
+msgid ""
+"Fence nodes that do not join the controller process group within this much "
+"time after joining the cluster, to allow the cluster to continue managing "
+"resources. A value of 0 means never fence pending nodes. Setting the value "
+"to 2h means fence nodes after 2 hours."
+msgstr ""
+"屏蔽加入群集后在此时间内未加入控制器进程组的节点，以便群集继续管理资源。"
+"值为0表示永远不屏蔽待处理节点。将值设为2h表示在2小时后屏蔽节点。"
```
Comment (gao-yan) [inline po/zh_CN.po:252]: So far the convention in all other places is keeping the term "fence" untranslated.

Besides, I realized both "集群" and "群集" are used for the term "cluster" as well in the existing translations. They'd better  be consistent eventually.

File: po/zh_CN.po [hunk 3]
```diff
@@ -961,127 +965,132 @@ msgstr ""
...
 
-#: tools/crm_resource.c:2028
+#: tools/crm_resource.c:2006
+msgid "You need to specify a resource type with -t"
+msgstr "需要使用-t指定资源类型"
+
+#: tools/crm_resource.c:2013
+#, fuzzy, c-format
+msgid "Could not delete resource %s: %s"
+msgstr "无法获得修改的CIB：%s\n"
```
Comment (gao-yan) [inline po/zh_CN.po:1086]: This translation doesn't really match.

**Review by gao-yan — COMMENTED** [review]
Thanks. Apart from Ken's point, a few comments in below.

File: po/zh_CN.po (hunk 3 above)
Comment (xuezhixin) [reply po/zh_CN.po]: OK, I will resubmit.

**Comment by wenningerk** [comment]
not that I can really help on such a topic - but I can press the button ;-)

File: po/zh_CN.po (hunk 2 above)
Comment (gao-yan) [reply po/zh_CN.po:252]: Thanks for the update.

Again, this is first time that "屏蔽" is being used for term the "fence". I'm not sure if it's the most clear or conventional translation. The thing is, it's not consistent with all the other places where "fence" all remains untranslated.

File: po/zh_CN.po [hunk 4]
```diff
@@ -234,6 +234,23 @@ msgstr ""
...
+"等待已加入集群的节点加入控制器进程组的时间"
+
+#: daemons/controld/controld_control.c:685 lib/pengine/common.c:166
+msgid ""
+"Fence nodes that do not join the controller process group within this much "
+"time after joining the cluster, to allow the cluster to continue managing "
+"resources. A value of 0 means never fence pending nodes. Setting the value "
+"to 2h means fence nodes after 2 hours."
+msgstr ""
+"Fence节点在加入集群后的一段时间内不加入控制器进程组，以便群集继续管理资源。"
```
Comment (gao-yan) [inline po/zh_CN.po:251]: How about:

"如果节点在加入集群后在此时间内不加入控制器进程组， Fence 该节点 ，以便 ..."

**Review by gao-yan — APPROVED** [review]
LGTM, thanks!

**Comment by kgaillot** [comment]
Thanks! I'll cherry-pick these into 2.1 for the 2.1.7 release.

**Pull request merged**